check-log/check-log.go

```diff
@@ -154,6 +164,9 @@ func (opts *logOpts) searchLog(logFile string) (int64, int64, string, error) {
 
 	f, err := os.Open(logFile)
 	if err != nil {
```

Comment:
It is not always file missing error. You should `continue` immediately when file is missing outsisde this function.